src/LiveComponent/src/DependencyInjection/LiveComponentExtension.php

```diff
@@ -48,7 +65,6 @@ public function load(array $configs, ContainerBuilder $container): void
             ->setArguments([
                 new TaggedIteratorArgument('twig.component.property_hydrator'),
                 new Reference('property_accessor'),
-                new Reference('annotation_reader'),
                 '%kernel.secret%',
             ])
         ;
```

Comment:
Below, for `LiveComponentSubscriber`, it's minor, but we could, I think, pass an abstract argument to arg 0, since it's filled in via the compiler pass.

Reply:
I think we would have the same problem as discussed [here](https://github.com/symfony/ux/pull/106#discussion_r659069501). How should I handle older versions?

Reply:
I say let’s use it (conditionally) if it’s not too much trouble.